JavaScript/JavaScriptSDK/appInsights.ts

```diff
@@ -190,16 +190,18 @@ module Microsoft.ApplicationInsights {
             if (!Telemetry.PageViewPerformance.isPerformanceTimingSupported()) {
                 // no navigation timing (IE 8, iOS Safari 8.4, Opera Mini 8 - see http://caniuse.com/#feat=nav-timing)
                 _InternalLogging.throwInternalNonUserActionable(LoggingSeverity.CRITICAL,
```

Comment:
I think this is about data quality (first assuring customer that page view will be sent, second letting customer know that no duration will be collected) - so I'd keep customers informed.
0 does not only mean that API is not supported - it could be API bug (it's supported but returns 0) or manual override, also different versions of SDK use API differently so it makes it harder to perform investigations about data quality issues. And specific errors simplify it.